tools/Deployment/config_gulp.json

```diff
@@ -22,8 +22,10 @@
     "masterUrl": "https://www.myget.org/F/docfx/api/v2/package"
   },
   "azdevops": {
-    "devUrl": "https://docfx.pkgs.visualstudio.com/docfx/_packaging/docs-build-v2-ppe/nuget/v3/index.json",
-    "masterUrl": "https://docfx.pkgs.visualstudio.com/docfx/_packaging/docs-build-v2-prod/nuget/v3/index.json"
+    "perfUrl": "https://docfx.pkgs.visualstudio.com/docfx/_packaging/docs-build-v2-perf/nuget/v3/index.json",
```

Comment:
@fenxu @superyyrrzz 
Shall we use v2 feed Urls ?
Sample: https://docfx.pkgs.visualstudio.com/docfx/_packaging/docs-build-v2-ppe/nuget/v2

Reply:
I think we should use v3 as it is latest API version and can work.